src/webui/www/private/scripts/client.js

```diff
@@ -73,7 +73,9 @@ window.qBittorrent.Client ??= (() => {
     };
 
     const getSyncMainDataInterval = () => {
-        return customSyncMainDataInterval ? customSyncMainDataInterval : serverSyncMainDataInterval;
+        return document.hidden
+            ? window.qBittorrent.Cache.preferences.get().web_ui_session_timeout * 500
```

Comment:
Could you remind me why the value is `500`?

Reply:
`seconds * 1000 / 2`, to convert to milliseconds, then half

Reply:
So the original must be `1000 / 2`. Then I would suggest:
```js
(window.qBittorrent.Cache.preferences.get().web_ui_session_timeout * 1000) / 2;  // halve the timeout
```